api/envoy/config/filter/http/ext_authz/v2/ext_authz.proto

```diff
@@ -109,6 +109,13 @@ message ExtAuthz {
   // When this field is true, Envoy will include the peer X.509 certificate, if available, in the
   // :ref:`certificate<envoy_api_field_service.auth.v2.AttributeContext.Peer.certificate>`.
   bool include_peer_certificate = 10;
+
+  // Specifies if HTTP status codes for gRPC calls reflect gRPC status
+  // https://github.com/envoyproxy/envoy/issues/11079
+  //
+  // When this field is true, Envoy will send an HTTP response code appropriate to
+  // the response seen
```

Comment:
Does this mean Envoy will send back the canonical HTTP status code for the observed gRPC status? i.e. by using `httpToGrpcStatus` from source/common/grpc/status.h?